plugins/woocommerce/includes/class-wc-regenerate-images.php

```diff
@@ -466,6 +504,79 @@ public static function queue_image_regeneration() {
 		// Lets dispatch the queue to start processing.
 		self::$background_process->save()->dispatch();
 	}
+
+	/**
+	 * Return the memory limit used for image processing in WP.
+	 *
+	 * @return int Number of bytes of available memory for the PHP process.
+	 */
+	protected static function get_image_memory_limit() {
```

Comment:
Hey, can we move this and `get_wp_image_memory_limit` function to somewhere in `src` and call from there via a container? that way, we will be able to test and mock them easily.

Reply:
Thanks, will do that after I'll get the confirmation that functionally it works (ie it stopped crashing).